salt/cloud/deploy/bootstrap-salt.sh

```diff
@@ -227,7 +227,7 @@ _SALT_MASTER_ADDRESS=${BS_SALT_MASTER_ADDRESS:-null}
 _SALT_MINION_ID="null"
 # _SIMPLIFY_VERSION is mostly used in Solaris based distributions
 _SIMPLIFY_VERSION=$BS_TRUE
-_LIBCLOUD_MIN_VERSION="0.14.0"
+_LIBCLOUD_MIN_VERSION="2.0.0"
```

Comment:
@vutny Is right. If changes are being made to the bootstrap script in salt, they will just be overwritten when we release a new bootstrap script. And changes to `bootstrap-salt.sh` need to be made in the [salt-bootstrap](https://github.com/saltstack/salt-bootstrap) repo if they are to persist after the next bootstrap version is released.